OpenRA.Game/Support/Log.cs

```diff
@@ -75,22 +77,33 @@ public static void AddChannel(string channelName, string baseFilename)
 			}
 		}
 
-		public static void Write(string channel, string value)
+		public static void Write(string channelName, string value)
 		{
-			var writer = Channel(channel).Writer;
+			var channel = Channels[channelName];
```

Comment:
Please change this to `var channel =  Channel(channel)`. The `Channel()` method uses locking to avoid race conditions and gives a more readable error message when someone tries to log to an undefined channel.

Reply:
Done.